docs/README.md

```diff
@@ -0,0 +1,65 @@
+# KILI SDK docs
```

Comment:
this should be part of the internal documentation in Notion right ?

Reply:
Oh yeah, this could be. I saw a lot of repositories having such a readme so I put the instructions here but you are right, we don't want people to feel like they can play with it.